tokenlists/typing.py

```diff
@@ -180,18 +184,24 @@ class Config:
         # NOTE: Not frozen as we may need to dynamically modify this
         froze = False
 
-    @validator("logoURI")
+    @field_validator("logoURI")
     def validate_uri(cls, v: Optional[str]) -> Optional[str]:
         if v is None:
             return v
 
-        if "://" not in v or not AnyUrl(v, scheme=v.split("://")[0]):
+        if "://" not in v or not AnyUrl(v):
             raise ValueError(f"'{v}' is not a valid URI")
 
         return v
 
-    def dict(self, *args, **kwargs) -> dict:
-        data = super().dict(*args, **kwargs)
-        # NOTE: This was the easiest way to make sure this property returns isoformat
-        data["timestamp"] = self.timestamp.isoformat()
+    def dict(self, *args, **kwargs):
+        return self.model_dump(*args, **kwargs)
```

Comment:
Do we want to remove this method and make this a breaking change PR? (I mean it probably should be a breaking change PR to mitigate any down-stream effects of v2-only dependency)

Reply:
Realizing that we have to remove this as it causes problems, removed here: [9683fe9](https://github.com/ApeWorX/py-tokenlists/pull/36/commits/9683fe91767dff1112196b9605f24b598fcd7434)

Reply:
And ya we can make these breaking change PRs